src/components/FSHControls.js

```diff
@@ -242,6 +337,50 @@ export default function FSHControls(props) {
           </Button>
         </DialogActions>
       </Dialog>
+      <Dialog
+        open={openExamples}
+        onClose={handleCloseExamples}
+        aria-labelledby="form-dialog-title"
+        maxWidth="lg"
+        fullWidth
+        scroll="paper"
+        classes={{ paper: classes.dialogPaper }}
+      >
+        <DialogTitle id="form-dialog-title">FSH Examples</DialogTitle>
+        <DialogContent>
+          <Grid className={classes.container} container>
+            <Grid item xs={4}>
+              <TreeView
+                className={classes.treeView}
+                defaultCollapseIcon={<ExpandMoreIcon />}
+                defaultExpanded={['root']}
+                defaultExpandIcon={<ChevronRightIcon />}
+                onNodeSelect={fetchExampleFSH}
+              >
+                {renderTree(props.exampleConfig)}
+              </TreeView>
+            </Grid>
+            <Grid item xs={8}>
+              <ExampleDisplayComponent
+                name={currentExample ? currentExampleName : ''}
+                value={currentExample}
+                initialText={currentExample}
+                updateTextValue={updateExampleValue}
+                mode={'fsh'}
+                placeholder={isFetchingExample ? 'Fetching example...' : 'Select an example'}
+              />
+            </Grid>
+          </Grid>
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={handleCopyToClipboard} color="primary">
+            <AssignmentOutlinedIcon></AssignmentOutlinedIcon> Copy to clipboard
```

Comment:
If it is significantly more difficult, I think it makes sense for practical reasons to leave this as "Copy to clipboard", but did you consider having a button you press that automatically loads the example into the editor? That is somewhat what I expected when I started playing with it.

Reply:
I actually did have that as a separate button, but removed it today after meeting with Mark. He thought that copying to clipboard was enough, and that the "send to main editor" option would just a potential source of confusion/frustration.

Reply:
Ok, that's fine with me. I agree two buttons would be confusing, and if Mark prefers a copy button to a "send to editor" button, that sounds good to me.